spyderlib/utils/icon_manager.py

```diff
@@ -181,10 +183,16 @@
     '1uparrow':                [('fa.angle-up',), {}],
     '2downarrow':              [('fa.angle-double-down',), {}],
     '1downarrow':              [('fa.angle-down',), {}],    
-    'pylint':                  [('fa.search', 'fa.check'), {'options': [{}, {'offset': (0.125, 0.125), 'color': 'orange'}]}],
     'attribute':               [('spyder.circle-letter-a',), {'color': 'magenta'}],
     'module':                  [('spyder.circle-letter-m',), {'color': '#daa520'}],
     'no_match':                [('fa.circle',), {'color': 'gray'}],
+    'no_match':                [('fa.circle',), {'color': 'gray'}],
+    # --- Third party plugins ------------------------------------------------
+    'profiler':                [('fa.clock-o',), {}],
+    'pylint':                  [('fa.search', 'fa.check'), {'options': [{}, {'offset': (0.125, 0.125), 'color': 'orange'}]}],
+    'condapackages':           [('fa.archive',), {}],
+    'spyplugins.ui.example':   [('fa.eye',), {}],
+    'spyplugins.ui.autopep8':  [('fa.eye',), {}],
```

Comment:
I still have not figured out this part.... we need to allow the extension of the qtawesome icons for third party plugins in an easy way. I think we need @SylvainCorlay on this